packages/nextjs/src/client/index.ts

```diff
@@ -27,6 +32,7 @@ export function init(options: BrowserOptions): void {
     environment: getVercelEnv(true) || process.env.NODE_ENV,
     defaultIntegrations: getDefaultIntegrations(options),
     ...options,
+    stackParser: options.stackParser || createStackParser(...[nextRoutesStackParser, ...defaultStackLineParsers]),
```

Comment:
What I think we should do is change the file so that we do:

```ts
import {
  defaultStackLineParsers as reactDefaultStackLineParsers,
  getDefaultIntegrations as getReactDefaultIntegrations,
  init as reactInit,
} from '@sentry/react';
// move next stack parser into @sentry/nextjs
import { nextRoutesStackParser } from './stack-parser.ts';

export const defaultStackLineParsers = [...reactDefaultStackLineParsers, nextRoutesStackParser];

export const defaultStackParser = createStackParser(...defaultStackLineParsers);
```

This way we override `defaultStackParser` for the users who create custom clients.